bakery/storage.go

```diff
@@ -52,10 +52,10 @@ func (s *memStorage) Get(location string) (string, error) {
 	defer s.mu.Unlock()
 	item, ok := s.values[location]
 	if !ok {
-		logf("storage.Get[%q] -> not found", location)
+		logger.Infof("storage.Get[%q] -> not found", location)
```

Comment:
Not sure. The error should be logged elsewhere anyway. It seems to me like the same level as the other storage access log messages is appropriate.